docs/recipes/WritingTests.md

```diff
@@ -79,7 +79,7 @@ export default function todos(state = initialState, action) {
   switch (action.type) {
   case ADD_TODO:
     return [{
-      id: (state.length === 0) ? 0 : state[0].id + 1,
+      id: (state.length === 0) ? 0 : state[state.length - 1].id + 1,
```

Comment:
I don't think it's a typo. The new item is appended to the beginning, so first ID is largest.
That said you can just replace it with `id: state.length`. Can can you change that please?